pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -86,6 +86,26 @@ type RequestedToCapacityRatioArgs struct {
 	Resources []ResourceSpec
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourcesLeastAllocatedArgs holds arguments used to configure NodeResourcesLeastAllocated plugin.
+type NodeResourcesLeastAllocatedArgs struct {
+	metav1.TypeMeta
+
+	// Resources to be managed
+	Resources []ResourceSpec
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourcesMostAllocatedArgs holds arguments used to configure NodeResourcesMostAllocated plugin.
+type NodeResourcesMostAllocatedArgs struct {
+	metav1.TypeMeta
+
+	// Resources to be managed
```

Comment:
This makes it sound like resources not listed here won't be considered. Does that happen?

Reply:
I don't have solid reason off my head to consider some factors we don't know yet.